tests/rules/pds3/test_coiss_xxxx.py

```
@@ -0,0 +1,326 @@
+# TODO: When cassini_iss_fring_mosaics_rsfrench2025 and
+# cassini_iss_spokes_hedman-hamilton-2024 bundles are available (e.g. in test
+# holdings), drop _coiss_opus_products_golden_references_pds4_reproj and the
+# test_opus_products / test_opus_id_to_primary_logical_path skips so PDS4 reproj
+# goldens are exercised again.
+_PDS4_REPROJ_BUNDLE_MARKERS = (
+    'cassini_iss_spokes_hedman-hamilton-2024',
+    'cassini_iss_fring_mosaics_rsfrench2025',
+)
+
+
+def _coiss_opus_products_golden_references_pds4_reproj(expected_relative):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_coiss_opus_products_golden_references_pds4_reproj`

(ANN202)